packages/plugin-vee-validate/src/index.js

```
@@ -119,6 +120,21 @@ export default function VeeValidatePlugin (opts) {
       parsedSchema: formSchemaWithVeeValidate
     }
   }
+
+  // extends the schema form props
+  const extend = ({ extendSchemaFormProps }) => {
+    extendSchemaFormProps({
+      validationSchema: {
+        type: null,
```

question: Shouldn't this be `object`?

---

I didn't want to console to panic whenever they pass `undefined` or `null`, **WDYT?**